src/OpenVRTrackersModule.cpp

```diff
@@ -87,6 +88,18 @@ bool OpenVRTrackersModule::configure(yarp::os::ResourceFinder& rf)
         tfRemote = rf.find("tfRemote").asString();
     }
 
+    // Try to find the "vrOrigin" entry
+    std::string vrOrigin;
+    if (!(rf.check("vrOrigin") && rf.find("vrOrigin").isString())) {
+        yInfo() << openvr_trackers_module::LogPrefix
+                << "Using default vrOrigin:"
+                << openvr_trackers_module::DefaultVrOrigin;
+        vrOrigin = openvr_trackers_module::DefaultVrOrigin;
+    }
+    else {
+        vrOrigin = rf.find("vrOrigin").asString();
+    }
```

Comment:
I realized that this could be error-prone. The user could spell it wrongly, or forget to use the first capital letter. In these cases, it is safer to keep the parsing logic in the configuration phase, while avoiding using open-ended string inputs in the interface.

Hence my suggestion would be to:
- take the string from configuration file and edit it to be all lowercase. You can see how to do that in https://en.cppreference.com/w/cpp/string/byte/tolower (see the notes). In this way the user can also avoid to remember the capital case.
- Move the if logic https://github.com/ami-iit/yarp-openvr-trackers/blob/2c1a682b99/src/OpenVRTrackersDriver.cpp#L121-L128 to the configuration part. The output should not be a string, but rather a value of https://github.com/ami-iit/yarp-openvr-trackers/blob/2c1a682b998a0d629495c96f3ded5661eff20e4a/src/OpenVRTrackersDriver.h#L25-L30. In this way you limit the possible values to only three, and you pass directly this value to the ``initialize`` method (and at this point the default value can be removed, because you set it every time)

Reply:
Agreed, done in https://github.com/ami-iit/yarp-openvr-trackers/pull/12/commits/ce9afa67ce093e6ee6a2f9ae920832a15141c832